microsoft-edge/webview2/release-notes.md

```diff
@@ -34,6 +34,225 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2EnvironmentOptions Class](#)
+* [CoreWebView2CustomSchemeRegistration Class](#)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2EnvironmentOptions4 interface](#)
+   * [ICoreWebView2EnvironmentOptions4::GetCustomSchemeRegistrations method](#)
+   * [ICoreWebView2EnvironmentOptions4::SetCustomSchemeRegistrations method](#)
+* [ICoreWebView2CustomSchemeRegistration interface](#)
+   * [ICoreWebView2CustomSchemeRegistration::GetAllowedOrigins method](#)
+   * [ICoreWebView2CustomSchemeRegistration::SetAllowedOrigins method](#)
+   * [ICoreWebView2CustomSchemeRegistration::HasAuthorityComponent property (get](#), [put)](#)
+   * [ICoreWebView2CustomSchemeRegistration::SchemeName property (get)](#)<!--no put-->
+   * [ICoreWebView2CustomSchemeRegistration::TreatAsSecure property (get](#), [put)](#)
+
+---
+
+<!-- ====================================================================== -->
+
+## 1.0.1671-prerelease
+
+Release Date: Feb 13, 2023  
+
+[NuGet package for WebView2 SDK 1.0.1671](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1671-prerelease)  
+
+For full API compatibility, this version of the WebView2 SDK requires Microsoft Edge version 112.0.1671.0 or higher. 
+
+### General
+
+#### Experimental features 
+
+*  Added support for the Experimental File API:
+
+##### [.NET/C#](#tab/dotnetcsharp)
+* [CoreWebView2ExperimentalFile interface](#)
+   * [CoreWebView2ExperimentalFile::get_Path method](#)
+* [CoreWebView2ExperimentalWebMessageReceivedEventArgs interface](#)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+* [CoreWebView2ExperimentalFile interface](#)
+   * [CoreWebView2ExperimentalFile::get_Path method](#)
+* [CoreWebView2ExperimentalWebMessageReceivedEventArgs interface](#)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2ExperimentalFile interface](#)
+   * [ICoreWebView2ExperimentalFile::get_Path method](#)
+* [ICoreWebView2ExperimentalWebMessageReceivedEventArgs interface](#)
+
+---
+
+*  Added support for Experimental Object Collection View API
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2ExperimentalObjectCollectionView interface](#)
+	* [CoreWebView2ExperimentalObjectCollectionView::get_Count method](#)
+	* [CoreWebView2ExperimentalObjectCollectionView:: GetValueAtIndex method](#)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2ExperimentalObjectCollectionView interface](#)
+	* [CoreWebView2ExperimentalObjectCollectionView::get_Count method](#)
+	* [CoreWebView2ExperimentalObjectCollectionView:: GetValueAtIndex method](#)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2ExperimentalObjectCollectionView interface](#)
+	* [ICoreWebView2ExperimentalObjectCollectionView::get_Count method](#)
+	* [ICoreWebView2ExperimentalObjectCollectionView:: GetValueAtIndex method](#)
+
+---
+
+#### Promotions
+
+The following APIs are promoted to stable in this prerelease SDK:
+
+*  The SharedBuffer API:

```

Comment:
```suggestion
###### The SharedBuffer API
```
we should try a heading instead of plain text.  Evidence: I need to find these, and I can't find them; as a plain-text list item they get lost.
In other files we're moving to h2/h4/h6 headings pattern, that's why I skipped h5 here.